kie-remote/src/main/java/org/kie/remote/impl/consumer/ListenerThread.java

```diff
@@ -36,11 +36,10 @@
 import org.kie.remote.message.ListKieSessionObjectMessage;
 import org.kie.remote.message.VisitableMessage;
 import org.kie.remote.message.VisitorMessage;
+import org.kie.remote.util.SerializationUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import static org.kie.remote.util.SerializationUtil.deserialize;
```

Comment:
Is this related to -Dfull changes?

Reply:
No, it is not probably some automatic IDE change during refactoring. Do you see it as a problem?

Reply:
It is not a problem but I would like to have an agreement about `static import` and similar otherwise it will become a "refactor" war in every PR :smile: 

I.e. we have many other import static in this repo:
`./kie-remote/src/main/java/org.kie.remote/impl/producer/EventProducer.java:import static org.kie.remote.util.SerializationUtil.serialize;
./hacep/src/main/java/org/kie/hacep/consumer/DroolsConsumerHandler.java:import static org.kie.remote.util.SerializationUtil.deserialize;
[...]`

Reply:
Btw it is a minor comment, considering that this PR is important to have master stable fine for me to merge and then we can discuss about that offline